pf4j-web-demo/demo/.gitignore

```diff
@@ -1 +0,0 @@
-/bin
```

Comment:
I think that it's a good idea to keep this file and to add other directories like:
```
target/
.classpath
.project
.settings
.idea/
*.iml
```

Reply:
On a multi module maven project, with nested directory structure it would be too many .gitignore files. would perfer to add them?

Reply:
My bad. I will remove it. My first impression was that we talk about `.gitignore` form the root of project.

Reply:
Thank you